packages/caravan-wallets/src/trezor.ts

```diff
@@ -804,6 +823,7 @@ export class TrezorSignMultisigTransaction extends TrezorInteraction {
   }
 }
 
+// eslint-disable-next-line multiline-comment-style
```

Comment:
Seems these changes (this and the two commented imports) can be removed

Reply:
Unsure why eslint called just this comment out. Didn't really want to dig into it and potentially muck up the diff even more. Maybe something about spacing?

Reply:
cleaned up a superfluous rule: [bbc5543](https://github.com/caravan-bitcoin/caravan/pull/41/commits/bbc554319ff886c0f1946b1cb5ed9213c2dfe0f5)